libs/wingc/src/lib.rs

```diff
@@ -131,6 +131,7 @@ pub fn parse(source_path: &Path) -> (Scope, Diagnostics) {
 		source_name: source_path.to_str().unwrap().to_string(),
 		error_nodes: RefCell::new(HashSet::new()),
 		diagnostics: RefCell::new(Diagnostics::new()),
+		is_json: RefCell::new(false),
```

Comment:
```suggestion
		in_json: RefCell::new(false),
```
maybe clearer (if you agree then probably relevant to all `is_json` fields as well)